lib/mongoid/locker/wrapper.rb

```diff
@@ -35,8 +35,8 @@ def self.find_and_lock(doc, opts)
                 { model.locked_at_field => { '$eq': nil } }
               ]
             },
-            {
-              '$where': "new Date() - this.#{model.locked_at_field} >= #{model.lock_timeout * 1000}"
+            { # The expr equals to "Time.now.utc >= model.locked_at_field + model.lock_timeout * 1000"
```

Comment:
Since you have to change CHANGELOG, I'll nitpick :) Put it on its own line. Lower the.

Reply:
Comment moved